<launch>
      <arg name="sensor_names" default="[echo1,echo2]"/>
      <node name="airsimnode_echos" pkg="airsim" type="airsimnode_echos.py" respawn="false" output="screen">

            <param name="ros_rate" type="int" value="10"/>
            <param name="topic_prefix" type="string" value="rtis"/>
            <param name="topic_suffix" type="string" value="pointcloud"/>
            <param name="vehicle_name" type="string" value="airsimvehicle" />
            <param name="frame_prefix" type="string" value="base_rtis"/>
            <param name="base_frame" type="string" value="world"/>
            <rosparam param="sensor_names" subst_value="True">$(arg sensor_names)</rosparam>
      </node>
</launch>